app/models/user.rb

```diff
@@ -1,3 +1,10 @@
 class User < ApplicationRecord
-	has_many :microposts
+  has_many :microposts
+  before_save {email.downcase!}
+  validates :name, presence: true, length: {maximum: Settings.valid.name_length}
+  VALID_EMAIL_REGEX = Settings.valid.email_syntax
+  validates :email, presence: true, length: {maximum: Settings.valid.email_length}, 
+  format: {with: VALID_EMAIL_REGEX}, uniqueness: {case_sensitive: false}
```

Comment:
Still I do not see the 2 space indentation for this line. You have to add this 2 space indentation as you break this line.